pandas/tests/io/test_parquet.py

```diff
@@ -41,20 +43,21 @@
     _HAVE_FASTPARQUET = False
 
 
-pytestmark = [
-    pytest.mark.filterwarnings("ignore:RangeIndex.* is deprecated:DeprecationWarning"),
-    # TODO(ArrayManager) fastparquet / pyarrow rely on BlockManager internals
-    td.skip_array_manager_not_yet_implemented,
-]
+pytestmark = pytest.mark.filterwarnings(
+    "ignore:RangeIndex.* is deprecated:DeprecationWarning"
+)
+
 
+# TODO(ArrayManager) fastparquet relies on BlockManager internals
 
 # setup engines & skips
 @pytest.fixture(
     params=[
         pytest.param(
             "fastparquet",
             marks=pytest.mark.skipif(
-                not _HAVE_FASTPARQUET, reason="fastparquet is not installed"
+                not _HAVE_FASTPARQUET or get_option("mode.data_manager") == "array",
+                reason="fastparquet is not installed or ArrayManager is used",
```

Comment:
is this a "for now" or a "ever"?

Reply:
If your question is about "will ArrayManager be supported with fastparquet engine", that's probably a question for the fastparquet package (and since this is only optional for now, there is still time to discuss that with them)

Reply:
ok, so not actionable on our end, thanks